data/hai missions.txt

```diff
@@ -1053,3 +1053,245 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+
+
+mission "Nanachi 1"
+	name "Escort Nanachi to <planet>"
+	description "Help Nanachi, a Hai who just bought her first ship, make a simple delivery to Cloudfire."
+	minor
+	source "Giverstone"
+	destination "Cloudfire"
+	deadline
+	to offer
+		"net worth" > 20000000
+		"combat rating" > 1200
+		year > 3018
+	on offer
+		conversation
+			`	As you are making your way through the spaceport you notice a worried-looking Hai staring at the local job board and back at some sort of document they are holding.`
+			choice
+				`	(Approach the Hai.)`
+				`	(Leave them alone.)`
+					decline
+			`	The Hai is a bit startled, but quickly calms down. "Sorry, you caught me by surprise. I've just bought my first ship and need to find some jobs to do, but nobody I've talked to wants to trust a beginner captain. I just don't know what to do anymore, I've been working for so long to qualify for this loan, but I'm running out of time and I'll never be able to pay it like this."`
+			`	Turns out the documents she is holding are the bank papers for the loan, somewhat similar to the one you took when you bought your first ship. You've come a long way since your days as a beginner, and know how hard it is to be starting out as a starship captain, you could probably convince someone here to offer a simple cargo job to this young Hai.`
+			choice
+				`	(Offer to help her find some jobs.)`
+				`	(Wish her luck and leave.)`
+					decline
+			`	Her ears twitch a bit, and her previously crestfallen eyes light up as she nearly jumps in joy at your offer. "Thank you so much captain! My name is Nanachi, I promise I won't cause any trouble at all, just a few quick jobs and I'm sure I'll be fine by myself!"`
+			`	"Captain <first> <last>. Can you show me your ship so I can look for jobs it's better suited for?"`
```

Comment:
You should indicate that this is the player speaking. This also gives me the same feel as a line in Zitchas' latest PR. Ref: https://github.com/endless-sky/endless-sky/pull/4222#discussion_r270873698. i.e. this seems like something that shouldn't be the player's dialog, instead something that is stated as having been said by the player, if that makes sense.
Perhaps change this line to something more like the following.
```suggestion
			`	You introduce yourself to Nanchi and shake her paw. "Can you show me your ship so I can look for jobs it's best suited for?" you ask.`
```